### PR TITLE
fix(server): re-enable prometheus counters.

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
Commit: 0213c08 ("tracee: add ready callback") made prometheus stats to stop being registered due to a timing issue. This patch re-enables it by registering prometheus counters in the callback function itself instead of in Tracee.New() (too early).

Fixes: #3303

> Still checking more stuff.

### CODE REVIEW

**Comment by geyslan** [comment]
> I'll redo the panels created in [#3297](https://github.com/aquasecurity/tracee/pull/3297#issuecomment-1620795046) on top of this working PR and raise the dashboard related one.

Ready: https://github.com/rafaeldtinoco/tracee/pull/7

**Comment by rafaeldtinoco** [comment]
Im adding an integration test to test if metrics are enabled and this will be ready.

**Comment by rafaeldtinoco** [comment]
So, if the "test-performance" target does not run (it has a skip if the user can't execute anything using "sudo -n" (with no password)) then I'll have to run the performance tests in a jenkins runner or change the runner config a bit. Lets see.

File: tests/testutils/tracee.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+type TraceeStatus int
+
+const (
+	TraceeStarted TraceeStatus = iota
+	TraceeFailed
+	TraceeTimedout
+	TraceeAlreadyRunning
+)
+
+// RunningTracee is a wrapper for a running tracee process as a regula process.
```
Comment (geyslan) [inline tests/testutils/tracee.go:37]:
```suggestion
// RunningTracee is a wrapper for a running tracee process as a regular process.
```

**Review by geyslan — APPROVED** [review]
LGTM

**Pull request merged**